trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1186,6 +1226,7 @@ impl<'a> CollapsableWitnessEntryTraverser<'a> {
     }
 
     // Inclusive.
+    #[allow(dead_code)]
     fn replace_next_n_entries_with_single_entry(&mut self, n: usize, entry: WitnessEntry) {
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
This one you can remove.